gensim/models/wrappers/fasttext.py

```diff
@@ -220,6 +220,10 @@ def save(self, *args, **kwargs):
         super(FastText, self).save(*args, **kwargs)
 
     @classmethod
+    def load_word2vec_format(cls, *args, **kwargs):
+        return FastTextKeyedVectors.load_word2vec_format(*args, **kwargs)
```

Comment:
I believe that a load using this method *only* learns the full-word vectors as in the `.vec` file. If so, isn't it true that the resulting object doesn't have any other capabilities beyond a plain `KeyedVectors`? In that case, using a specialized class like `FastTextKeyedVectors` – that maybe is trying to do more, such as ngram-tracking, but inherently is not because that info was lost in the sequence-of-steps used to load it – seems potentially misleading. So unless I'm misunderstanding, I think this load-technique should use a plain `KeyedVectors`.